client/signup/config/flows.js

```diff
@@ -53,6 +53,24 @@ const flows = {
 		description: 'Create an account and a blog and then add the business plan to the users cart.',
 		lastModified: '2016-01-21'
 	},
+	
+	businessv2: {
+		steps: ['domains', 'user' ],
```

Comment:
Super minor style note, but this array is missing a leading space after the `[`:

```
steps: [ 'domains', 'user' ]
```